scripts/build_data.py

```
@@ -208,37 +236,64 @@ def refresh_raw_sources(
-            write_json(raw_path, scrape_google_list_url(source, headed=headed))
+        failures: list[str] = []
+        for source, raw_path, backup_available in refresh_jobs:
+            try:
+                payload = scrape_google_list_url_with_retries(
+                    source,
+                    headed=headed,
+                    refresh_retries=refresh_retries,
+                    refresh_retry_backoff_seconds=refresh_retry_backoff_seconds,
+                    refresh_startup_jitter_seconds=effective_startup_jitter_seconds,
+                )
+            except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)